openlibrary/macros/QueryCarousel.html

```diff
@@ -19,7 +19,7 @@
   </form>
 
 $code:
-  params = { 'q': query }
+  params = { 'q': query, 'fields': '*,availability' }
```

Comment:
Ensures the "load more" solr query behind the scene also fetches / shuffles in book availability for carousels